src/middlewared/middlewared/plugins/kubernetes_linux/yaml.py

```diff
@@ -1,5 +1,27 @@
 import yaml
 
+class NoDatesFullLoader(yaml.FullLoader):
```

Comment:
Also let's please rename this to something which is more or less reflecting reality which means that dates are not being removed but just used as strings